transports/sqs/delayed-delivery_enable_sqs_[6,).partial.md

```diff
@@ -0,0 +1 @@
+In version 6 and higher the unrestricted delayed delivery feature is always enabled but in order to work it has to be enabled in both the sender and the receiver. When communicating with endpoints running older version of the transport make sure to ensure it is enabled.
```

Comment:
```suggestion
In version 6 and higher the unrestricted delayed delivery feature is always enabled by default. In order to work, it has to be enabled in both the sender and the receiver. When communicating with endpoints running older version of the transport make sure to ensure it is enabled.
```

> When communicating with endpoints running older version of the transport make sure to ensure it is enabled.

That sentence reads a bit weird to me. Why shouldn't it be enabled anyway if I want to make use of it? Why is the fact that communicating an older endpoint relevant when I'm supposed to have it enabled anyway?